### PR TITLE
Sort versions as dates not as strings

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

**Comment by cjcolvar** [comment]
:+1:

**Comment by hectorcorrea** [comment]
Looks good to me.

File: lib/active_fedora/versions_graph.rb [hunk 1]
```diff
@@ -2,11 +2,11 @@ module ActiveFedora
...
     def all opts={}
-      if opts[:include_auto_save]
-        fedora_versions
-      else
-        fedora_versions.reject { |v| v.label.match("auto") }
+      versions = fedora_versions
+      unless opts[:include_auto_save]
+        versions.reject! { |version| version.label.match("auto") }
       end
+      versions.sort_by { |version| DateTime.parse(version.created) }
```
Comment (hectorcorrea) [inline lib/active_fedora/versions_graph.rb]: What happens here is one `version.created` cannot be parsed as a valid date?

Comment (mjgiarlo) [inline lib/active_fedora/versions_graph.rb]: :bomb:

Comment (jcoyne) [inline lib/active_fedora/versions_graph.rb]: It seems reasonable to raise an exception if Fedora is giving us a non-date here.

Comment (mjgiarlo) [inline lib/active_fedora/versions_graph.rb]: Fixed in latest.

**Comment by hectorcorrea** [comment]
:+1: Waiting on Travis....

**Pull request merged**